core/string/src/str.rs

```diff
@@ -191,6 +191,21 @@ impl<'a> JsStr<'a> {
         I::get(self, index)
     }
 
+    /// Returns an element or subslice depending on the type of index, without doing bounds check.
```

Comment:
This seems like a better doc comment than the one used for `JsString`s version of the method. Can you reuse this for that?